lib/datawarehouse/index.ts

```diff
@@ -146,9 +185,12 @@ export interface DataWarehouseArgs {
     isDev?: boolean;
 }
 
+export type DataFormat = "JSON" | "parquet";
```

Comment:
Is this the check that DataFormat is either JSON or parquet? Or does that happen in line 193: dataFormat?: DataFormat;

Reply:
this is a form of union type called a [string literal type](https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types). Using `DataFormat` as a function parameter type ensures that a compile time error will be thrown if anything else is passed in other than those two strings.